chainercv/visualizations/vis_label.py

```diff
@@ -64,6 +65,11 @@ def vis_label(
             value is :obj:`0`, the figure will be completely transparent.
             The default value is :obj:`1`. This option is useful for
             overlaying the label on the source image.
+        all_label_names_in_legend (bool): Determines whether to include
+            all label names in a legend. If this is :obj:`False`,
+            only the names of labels
+            that are included in the label image are included.
```

Comment:
In this sentence, ` are included` is used twice. It is hard to read. How about `If this is :obj:False, the legend does not contain the names of unused labels in legend. A unused label is defined as a label that does not appear in :obj:label.`